Makefile

```diff
@@ -1,5 +1,17 @@
 .PHONY: all doc doc-man doc-latex docclean
```

Comment:
PHONY variable need alignment with new targets introduce by this commit.
Please check https://www.gnu.org/software/make/manual/html_node/Phony-Targets.html

Whereas this link highlight the aim of phony rule, it also explain a potential issue while looping over directory to call recursive make as done below.
I checked quickly, but this might apply to this commit as well : if I understand correctly, the issue could be a sub-make failing while encountering a build error, but treated as a successful build by the parent make.

Reply:
Good catch @Yanok35.
I followed the advices from the link you provided by declaring the sub-directories as .PHONY targets.

As several targets could depend on $(pfs), I used a variable to pass the "sub make" target:
```
distclean: PF_TARGET = distclean
distclean: $(pfs)

$(pfs):
	$(MAKE) -j$$(nproc) -C $@ $(PF_TARGET)
```
https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html

It seems functionnal but what do you think about this ?